ftw/tabbedview/tests/test_caching.py

```diff
@@ -0,0 +1,24 @@
+from ftw.tabbedview.testing import TABBEDVIEW_FUNCTIONAL_TESTING
+from plone.app.caching.interfaces import IETagValue
+from unittest2 import TestCase
+from zope.component import getMultiAdapter
+
+
+class TestETagValue(TestCase):
+    layer = TABBEDVIEW_FUNCTIONAL_TESTING
+
+    def setUp(self):
+        self.portal = self.layer['portal']
+        self.request = self.layer['request']
+
+    def test_etag_value_returns_default_tab(self):
```

Comment:
This actuall tests on the portal, proving that it does not depend on the portal type.